dataprofiler/profilers/profiler_options.py

```diff
@@ -859,9 +860,12 @@ def __init__(self, is_enabled=True, is_case_sensitive=True,
         super().__init__(is_enabled=is_enabled)
         self.is_case_sensitive = is_case_sensitive
         self.stop_words = stop_words
+        self.num_most_common_words = num_most_common_words
+        self.num_most_common_chars = num_most_common_chars
```

Comment:
to match the categorical, can we call these variables `top_k_words` and `top_k_chars`?

Reply:
changed